src/modules/rc_update/RCUpdateTest.cpp

```diff
@@ -0,0 +1,224 @@
+/****************************************************************************
+ *
+ *   Copyright (C) 2019 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (C) 2022 PX4 Development Team. All rights reserved.
```

Reply:
Fixed by interactive rebase in first commit.